test/object-store/sync/app.cpp

```diff
@@ -1295,15 +1294,16 @@ class UnitTestTransport : public GenericNetworkTransport {
                             {"data", profile_0}})
                 .dump();
 
-        completion_block(Response{.http_status_code = 200, .custom_status_code = 0, .headers = {}, .body = response});
+        Response resp{200, 0, {}, response};
+        completion_block(resp);
```

Comment:
Not a big deal since this is test code, but either this should go back to a single expression like `completion_block(Response{200, 0, {}, response});` (my personal style preference) or it should `std::move(resp)`. Ditto elsewhere

Reply:
Made inline again.